---
title: "CLASSIFICATION_WITH_SILVA_REFERENCES"
author: "Corine Jackman"
date: "April 27, 2016"
output: word_document
---
###Assignment

Re-classify the sequences  used using either the greengenes or SILVA references. The assignment should be submitted as an Rmd document in a repository on GitHub. You should supply the code you used to run mothur, but this does not need to be run when you use knitr to convert the document from Rmd to html. All of the R code that you used should be embedded in the Rmd document.

###Approach

The Silva reference was used for classifcation of OTUs and removal  of undesired reads. In order to use the Silva reference for classification, the classify.seq command was used. In the count file, reference and taxonomy files were replaced with **reference=silva.bacteria.fasta** and **taxonomy="silva.bacteria.silva.tax** as opposed to **trainset9_032012.pds.fasta** and**taxonomy=trainset9_032012.pds.tax** as used in class. These commands generated the following files: tree.sum, tree.train, silva.wang.tax.summary and silva.wang.taxonomy.

To use the remove.lineage for a Silva reference, the term **pbs** was replaced with **silva** The command generated the follwing files: pick.pick.fasta, pick.silva.wang.pick.taxonomy and pick.pick.count_table.

The process of removing chimeras can also be carried out with a silva-based alignment using chimera.slayer but was not used in this assignment (noted at http://www.mothur.org/wiki/Silva_reference_files under Release 123.)

###Code Chunks
In order to classify the sequences the following code was used.

```{r}
classify.seqs(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.fasta, count=stability.trim.contigs.good.unique.good.ilter.unique.precluster.denovo.uchime.pick.count_table, reference=silva.bacteria.fasta, taxonomy=silva.bacteria.silva.tax, cutoff=80)
```

To remove undesired sequences, the following code was used.
```{r}
remove.lineage(fasta=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.fasta, count=stability.trim.contigs.good.uunique.good.filter.unique.precluster.denovo.uchime.pick.count_table, taxonomy=stability.trim.contigs.good.unique.good.filter.unique.precluster.pick.silva.wang.taxonomy, taxon=Chloroplast-Mitochondria-unknown-Archaea-Eukaryota)
```

###Advantages to SILVA
The Silva Release 123 is 50,000 columns long, enabling compatibility with 16S rRNA and 18S rRNA sequences. However, in this study, bacteria was only accounted for.  
